catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/query/handlers/CqlTransformHandlerTest.java

```diff
@@ -78,7 +78,7 @@
   private static final String SERVICE_NOT_FOUND = "{\"message\":\"Service not found\"}";
   private static final String SERVICE_SUCCESS = Boon.toJson("");
   private static final String ATTACHMENT_REGEX =
-      "^attachment;filename=export-\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d{3}Z."
+      "^attachment;filename=export-\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d{1,3}Z."
```

Comment:
in the CI case we see here https://jenkins.codice.org/job/DDF-Jobs/job/pr/job/Linux/5487/
not sure if that example with `2018-10-11T17:34:55Z.kml` would work since theres no .xxxZ at the end. i think making the last `.` and `\\d{1,3}` optional with the `?` appended to them

Reply:
kudos to you for finding that 1/1000 chance of error though

Reply:
You're right. Thank you. I'll update this 👍